enioka_scan/src/main/java/com/enioka/scanner/sdk/zebraoss/ssi/SsiMonoPacket.java

```diff
@@ -0,0 +1,157 @@
+package com.enioka.scanner.sdk.zebraoss.ssi;
+
+/**
+ * SSI packet, containing the header and checksum bytes. Suitable for both incoming and outgoing packets.
+ * Multi-packets are (so far) read-only and in a separate class as they require extra header elements.
+ */
+public final class SsiMonoPacket {
+    private static final String LOG_TAG = "SsiMonoPacket";
+    private final boolean ble;
+
+    // Packet attributes are in the same order as which they occur in the raw packet buffer
+    /** MSB of total length in little-endian used by BLE only, for outgoing packets only (these bytes should be skipped for reading). */
```

Comment:
why skipped? Aren't they useful to detect if reading is complete?

Reply:
No longer used